### PR TITLE
Update vscode.rst

### PR DESCRIPTION
**Pull request by Ernestopheles**
Made the documentation appropriate to version 3 of xdebug

# update for xdebug version 3.x

### CODE REVIEW

**Comment by fkaminski** [comment]
What about Windows User here?

File: docs/intermediate/configure-php-xdebug/linux/vscode.rst [hunk 1]
```diff
@@ -125,18 +125,17 @@ Copy/paste all of the following lines into the above created ``xdebug.ini`` file
...
+      :emphasize-lines: 3,5
+
+      xdebug.mode               = debug
+      xdebug.start_with_request = yes
+      xdebug.client_host        = docker.for.lin.localhost
+      xdebug.client_port        = 9003
+      xdebug.idekey             = VSCODE
+      
+      ; Optional: If you want to have a log written
+      xdebug.log                = /shared/httpd/mytest/htdocs/xdebug.log
```
Comment (cytopia) [inline docs/intermediate/configure-php-xdebug/linux/vscode.rst:137]: This is very project specific. How would you configure it for all projects?

File: docs/intermediate/configure-php-xdebug/linux/vscode.rst [hunk 2]
```diff
@@ -78,7 +78,7 @@ You will need to configure the path mapping in ``launch.json`` (VSCode configura
                   "name": "Xdebug for Project mytest",
                   "type": "php",
                   "request": "launch",
-                  "port": 9000,
+                  "port": 9003,
```
Comment (cytopia) [inline docs/intermediate/configure-php-xdebug/linux/vscode.rst:81]: Why does this need to be changed?

**Comment by cytopia** [comment]
@Ernestopheles it seems to have been covered with this PR already: https://github.com/cytopia/devilbox/pull/877/files

**Pull request closed without merging**